runtime/ptrace_ios.h

```diff
@@ -0,0 +1,18 @@
+// Copyright 2019 The Flutter Authors. All rights reserved.
```

Comment:
All our licenses have the 2013 copyright date.

Reply:
Done.